qt/src/DisplayRenderer.hh

```diff
@@ -23,6 +23,8 @@
 #include <QString>
 #include <QMutex>
 
+#include "Kdjvu.hh"
```

Comment:
You can forward declare the class instead and include the header only in the source file.

Reply:
Is it problem for gImageReader? Do you want reduce compile time, right?

Reply:
Well it's general good practice in c++ to only include things when you really need to. This particular instance will have a limited effect, but if you start doing it everywhere you will definitely notice. So easier to just always follow the good practice.

Reply:
Ok, I understood you point of view ;-)